lib/server/app.js

```diff
@@ -26,8 +26,6 @@ function create (env, ctx) {
   var appInfo = env.name + ' ' + env.version;
   app.set('title', appInfo);
   app.enable('trust proxy'); // Allows req.secure test on heroku https connections.
-  app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
These lines enforce a 100Kb limit for the entire server, regardless of per route configurations.  Removing these lines allows the routes to properly express their size limit policies.